include/zephyr/bluetooth/gatt.h

```diff
@@ -1050,6 +1053,9 @@ static inline int bt_gatt_notify(struct bt_conn *conn,
 	params.attr = attr;
 	params.data = data;
 	params.len = len;
+#if defined(CONFIG_BT_EATT)
+	params.chan_option = BT_ATT_CHAN_ANY;
+#endif /* CONFIG_BT_EATT */
```

Comment:
I think this is probably the best approach to make the change as un-disruptive as possible :)

Reply:
Yes, the behavior should be the same as before. I tried to find all usages in the tree, but I may have missed some.